en/V3-Software_Platform_Requirements.md

```diff
@@ -17,10 +17,10 @@ The Linux operating system is one of the most popular in IoT. It has many featur
 | **3.1.1** | Verify that the bootloader does not allow code to be loaded from arbitrary locations. Locations include both storage (SD, USB, etc.) and network locations (over TCP/IP). | ✓ | ✓ | ✓ |
 | **3.1.2** | Verify bootloader configurations are immutable in production releases. | | ✓ | ✓ |
 | **3.1.3** | Verify that communication interfaces such as, USB, UART, and other variants are disabled or adequately protected during every stage of the device's boot process. | | ✓ | ✓ |
-| **3.1.4** | Verify that the authenticity of the first stage bootloader is verified by a trusted component of which the configuration in read-only memory (ROM) cannot be altered. e.g. CPU Based Secure Boot/Trusted Boot | | ✓ | ✓ |
-| **3.1.5** | Verify that the authenticity of next bootloader stages or application code is cryptographically verified during every step of the boot process. | | ✓ | ✓ |
+| **3.1.4** | Verify that the authenticity of the first stage bootloader is verified by a trusted component of which the configuration in read-only memory (ROM) cannot be altered. e.g. CPU Based Secure Boot/Trusted Boot/Hardware Trust Anchor (HTA) | | ✓ | ✓ |
+| **3.1.5** | Verify that the authenticity of next bootloader stages or application code is cryptographically verified before execution of every step of the boot process. | | ✓ | ✓ |
 | **3.1.6** | Verify that bootloader stages do not contain sensitive information (e.g. private keys or passwords logged to the console) as part of device start-up.  | | ✓ | ✓ |
-| **3.1.7** | Verify that firmware is stored in an encrypted volume at rest. | | ✓ | ✓ |
+| **3.1.7** | Verify that firmware is stored in an authenticated volume at rest. | | ✓ | ✓ |
```

Comment:
In order to safeguard the confidentiality of the firmware, we really do want encryption here. 
3.1.5 covers authenticity of bootloader stages and application code (including firmware). Perhaps we should rephrase 3.1.5 to mention firmware specifically?

Reply:
I understand the want of confidentiality, although in some safety-critical systems or low-resource environments (such as automotive ECUs) having encrypted volumes is not the chosen approach.  I recommend to take that into account in the platform requirements if those type of devices are still to be considered IoT and we want it to be actionable and not just aspirational for them.